src/routes/event.js

```diff
@@ -1,15 +1,17 @@
-import { Router } from 'express';
-import asyncHandler from '../middlewares/asyncHandler';
-import eventController from '../controllers/event';
-import { auth, adminAuth } from '../middlewares/authorization';
+import { Router } from "express";
+import asyncHandler from "../middlewares/asyncHandler";
+import eventController from "../controllers/event";
+import { auth, adminAuth } from "../middlewares/authorization";
+import { getSignedUrl } from "../controllers/files";
 
 
 const eventRouter = new Router();
 
 eventRouter
-    .post('/', [auth, adminAuth], asyncHandler(eventController.postEvent))
-    .get('/', [auth, adminAuth], asyncHandler(eventController.getEvents))
-    .delete('/:id', [auth, adminAuth], asyncHandler(eventController.deleteEvent))
-    .put('/:id', [auth, adminAuth], asyncHandler(eventController.updateEvent))
+    .post("/", [auth, adminAuth], asyncHandler(eventController.postEvent))
+    .get("/signedUrl", [auth, adminAuth], asyncHandler(getSignedUrl))
+    .get("/", [auth, adminAuth], asyncHandler(eventController.getEvents))
```

Comment:
This route should not be protected. I think everyone should be able to access a list of all events, unless if we want to have some restrictions on which events users can see, and which ones they cannot. 

It would be nice if we can get one event too.